lib/bundle/locker.rb

```diff
@@ -132,6 +134,15 @@ def mas_list
       end
     end
 
+    def whalebrew_list
+      @whalebrew_list ||= begin
+        Bundle::WhalebrewDumper.images.each_with_object({}) do |image, name_versions|
+          _, version = `docker image inspect #{image} --format '{{ index .RepoDigests 0 }}'`.split(':')
```

Comment:
```suggestion
          , version = `docker image inspect #{image} --format '{{ index .RepoDigests 0 }}'`.split(':')
```

think that will work